_protected/app/langs/es_ES/language.php

```diff
@@ -0,0 +1,345 @@
+    'it' => 'Italiano',
+    'pt' => 'Português',
+    'pt' => 'Português',
+    'ru' => 'русский',
+    'sa' => 'العربية',
+    'he' => 'עִבְרִית',
+    'jp' => '日本語',
+    'cn' => '中文简体',
+    'cn' => '中文 繁體',
+    'tr' => 'Türkçe',
+    'dk' => 'Dansk',
+    'no' => 'Norsk',
+    'cz' => 'Český',
+    'bn' => 'বাংলা',
+    'ro' => 'Română',
+    'pl' => 'Polski',
+    'ba' => 'Bosanski',
+    'hr' => 'Hrvatski',
+    'rs' => 'Srpski',
+    'si' => 'Slovenski',
+
+    /***** Time Zone *****/
+    '-12' => '(UTC -12:00) Eniwetok, Kwajalein',
+    '-11' => '(UTC -11:00) Midway Island, Samoa',
+    '-10' => '(UTC -10:00) Hawaii',
+    '-9' => '(UTC -9:00) Alaska',
+    '-8' => '(UTC -8:00) Pacific Time (US & Canada)',
+    '-7' => '(UTC -7:00) Mountain Time (US & Canada)',
+    '-6' => '(UTC -6:00) Central Time (US & Canada), Mexico City',
+    '-5' => '(UTC -5:00) Eastern Time (US & Canada), Bogota, Lima',
+    '-4' => '(UTC -4:00) Atlantic Time (Canada), Caracas, La Paz',
+    '-3.5' => '(UTC -3:30) Newfoundland',
+    '-3' => '(UTC -3:00) Brazil, Buenos Aires, Georgetown',
+    '-2' => '(UTC -2:00) Mid-Atlantic',
+    '-1' => '(UTC -1:00 hour) Azores, Cape Verde Islands',
+    '+0' => '(UTC) Western Europe Time, London, Lisbon, Casablanca',
+    '+1' => '(UTC +1:00 hour) Brussels, Copenhagen, Madrid, Paris',
+    '+2' => '(UTC +2:00) Kaliningrad, South Africa',
+    '+3' => '(UTC +3:00) Baghdad, Riyadh, Moscow, St. Petersburg',
+    '+3.5' => '(UTC +3:30) Tehran',
+    '+4' => '(UTC +4:00) Abu Dhabi, Muscat, Baku, Tbilisi',
+    '+4.5' => '(UTC +4:30) Kabul',
+    '+5' => '(UTC +5:00) Ekaterinburg, Islamabad, Karachi, Tashkent',
+    '+5.5' => '(UTC +5:30) Bombay, Calcutta, Madras, New Delhi',
+    '+6' => '(UTC +6:00) Almaty, Dhaka, Colombo',
+    '+7' => '(UTC +7:00) Bangkok, Hanoi, Jakarta',
+    '+8' => '(UTC +8:00) Beijing, Perth, Singapore, Hong Kong',
+    '+8.75' => '(UTC +8:45) Eucla',
+    '+9' => '(UTC +9:00) Tokyo, Seoul, Osaka, Sapporo, Yakutsk',
+    '+9.5' => '(UTC +9:30) Adelaide, Darwin',
+    '+10' => '(UTC +10:00) Eastern Australia, Guam, Vladivostok',
+    '+10.5' => '(UTC +10:30) Lord Howe Island',
+    '+11' => '(UTC +11:00) Magadan, Solomon Islands, New Caledonia',
+    '+11.5' => '(UTC +11:30) Norfolk Island',
+    '+12' => '(UTC +12:00) Auckland, Wellington, Fiji, Kamchatka',
+    '+12.75' => '(UTC +12:45) Chatham Islands',
+    '+13' => '(UTC +13:00) Phoenix Islands, Tonga',
+    '+14' => '(UTC +14:00) Line Islands',
+
+    /***** GENDER *****/
+    'male'=>'Man',
+    'female'=>'Woman',
+    'couple'=>'Couple',
+    'male,female'=>'Men and Women',
+    'male,couple'=>'Men and Couple',
+    'female,couple'=>'Women and Couple',
+    'male,female,couple'=>'Men, Women and Couple',
+
+    /***** BYTE UNIT *****/
+    'byte' => '%0% Bytes',
+    'kilobyte' => '%0% KB',
+    'megabyte' => '%0% MB',
+    'gigabyte' => '%0% GB',
+    'terabyte' => '%0% TB',
+
+    /***** USER'S INFO FIELDS *****/
+    'address' => 'Address:',
```

Comment:
FYI, those strings are used in the custom user fields. If you want to have them in Spanish, they should be translated as well